pkgs/development/zig-packages/build-zig-package/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, zig }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, enableParallelBuilding ? true
+, meta ? { }
+, ...
+}@attrs:
+
+stdenv.mkDerivation (attrs // {
+  inherit strictDeps enableParallelBuilding;
+  nativeBuildInputs = [ zig ] ++ nativeBuildInputs;
+
+  postConfigure = ''
+    export HOME=$TMPDIR
+  '';
+
+  buildPhase = attrs.buildPhase or ''
+    runHook preBuild
+    zig build -Drelease-safe -Dcpu=baseline
```

Comment:
This command line should be user-configurable (with a default value). Something like `zigBuildFlags = attrs.zigBuildFlags or [ . . . ];`

Reply:
Done